styles.css

```diff
@@ -3,6 +3,7 @@
   font-family: 'Roboto', sans-serif;
   color: #fff;
   text-decoration: none;
```

Comment:
- [ ] The * is an initial for the page like the width and height  are auto, and I think they are causing error in the display.